LICENSE

```diff
@@ -2,6 +2,8 @@ The MIT License (MIT)
 
 Copyright (c) 2015 Sebastian Kreutzberger
 
+Copyright 2025 MFB Technologies, Inc.
```

Comment:
I'd move this above the original copyright, e.g., at line 3 and change it to say

Updates from 2025 on are copyright MFB Technologies, Inc.